packages/infrastructure/webpack/src/get-react-config.js

```diff
@@ -1,13 +1,16 @@
 const getNodeConfig = require("./get-node-config");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const path = require("path");
+
 
 module.exports =
-  ({ miniCssExtractPluginLoader = MiniCssExtractPlugin.loader } = {}) =>
+  ({ miniCssExtractPluginLoader = MiniCssExtractPlugin.loader, dependencies = { resolvePath: path.resolve, scriptDirectory: __dirname } } = {}) =>
```

Comment:
Is `scriptDirectory` as `__dirname` local to package `@k8lens/webpack`? Not sure how that works. Let's talk? :)

Reply:
I guess it's **local** variable inside exported function? But why to name it as **global** `__dirname`?

Reply:
It's pretty much taken from https://github.com/lensapp/lens/blob/master/packages/open-lens/webpack/vars.ts#L19 

Where it's also `__dirname`. It's inside the function so that we can overwrite it in a unit test to point somewhere else. Otherwise the unit test will depend on the machine it is run and give a path like `/Users/jheikka/code/lens/packages/webpack` and so on.

Do you have a better idea how to do this?

Reply:
this has been removed